src/utils/forms.py

```diff
@@ -144,8 +144,12 @@ def text_sanitizer(text_value, tags=None, attrs=None, excl=ENTITIES_MAP):
 
 def plain_text_validator(value):
     """ A field validator that ensures a textual input has no harmful code"""
-    sanitized = text_sanitizer(value)
-    if value != sanitized:
+
+    # bleach strips carriage returns, convert them to newlines.
+    cleaned = value.replace("\r", "")
```

Comment:
The comment is inaccurate, we are not converting them to new lines. In this case it doesn't matter what we convert them to, all we want to do is for them to be ignored for validation.

Reply:
I may be missing something but I believe this will not work for most cases, because it is not handling POSIX `\n`.

https://en.wikipedia.org/wiki/Newline#Representation

https://github.com/django/django/blob/52116774549e27ac5d1ba9423e2fe61c5503a4a4/django/template/defaultfilters.py#L485-L494

Would be great if for clarity we can also use a variable name like `string_with_no_newlines` rather than `cleaned`. The same name `cleaned` is also used above in `text_sanitizer` to mean something different, and `'cleaned' == 'sanitized'` is hard to read anyway.

Reply:
Ah, that's what I thought it was initially and then second-guessed. So yeah, a better variable name would be great.